### ISSUE TITLE
[install error] sdotsub.f:(.text+0x7): undefined reference to `sdot_'

### ISSUE THREAD

**Issue by lhdgriver** [dmlc/mxnet#1067]
Linux + blas

I have put libblas.a  libcblas.a to /usr/local/lib, when I make, the cmd complains:
outside_built/gcc4.9.0/bin/g++ -DMSHADOW_FORCE_STREAM -Wall -O3 -I./mshadow/ -I./dmlc-core/include -fPIC -Iinclude -msse3 -funroll-loops -Wno-unused-parameter -Wno-unknown-pragmas -DMSHADOW_USE_CUDA=0 -DMSHADOW_USE_CBLAS=1 -DMSHADOW_USE_MKL=0 -DMSHADOW_RABIT_PS=0 -DMSHADOW_DIST_PS=0 -DMXNET_USE_OPENCV=1 `pkg-config --cflags opencv` -fopenmp   -o bin/im2rec tools/im2rec.cc build/resource.o build/c_api/c_api.o build/c_api/c_api_error.o build/c_api/c_predict_api.o build/common/mxrtc.o build/common/tblob_op_registry.o build/engine/engine.o build/engine/naive_engine.o build/engine/threaded_engine.o build/engine/threaded_engine_perdevice.o build/engine/threaded_engine_pooled.o build/io/io.o build/io/iter_csv.o build/io/iter_image_recordio.o build/io/iter_mnist.o build/kvstore/kvstore.o build/ndarray/ndarray.o build/ndarray/ndarray_function.o build/ndarray/unary_function.o build/operator/activation.o build/operator/batch_norm.o build/operator/block_grad.o build/operator/concat.o build/operator/convolution.o build/operator/cross_device_copy.o build/operator/cudnn_batch_norm.o build/operator/deconvolution.o build/operator/dropout.o build/operator/elementwise_binary_op.o build/operator/elementwise_binary_scalar_op.o build/operator/elementwise_sum.o build/operator/embedding.o build/operator/fully_connected.o build/operator/identity_attach_KL_sparse_reg.o build/operator/leaky_relu.o build/operator/lrn.o build/operator/native_op.o build/operator/ndarray_op.o build/operator/operator.o build/operator/pooling.o build/operator/regression_output.o build/operator/reshape.o build/operator/slice_channel.o build/operator/softmax_activation.o build/operator/softmax_output.o build/operator/swapaxis.o build/operator/upsampling.o build/optimizer/optimizer.o build/optimizer/sgd.o build/storage/storage.o build/symbol/graph_executor.o build/symbol/static_graph.o build/symbol/symbol.o dmlc-core/libdmlc.a -pthread -lm -lcblas -lrt -lcblas `pkg-config --libs opencv` 
/usr/local/lib/libcblas.a(sdotsub.o): In function `sdotsub_':
sdotsub.f:(.text+0x7): undefined reference to `sdot_'
/usr/local/lib/libcblas.a(cblas_sgemm.o): In function `cblas_sgemm':
cblas_sgemm.c:(.text+0x13c): undefined reference to `sgemm_'
cblas_sgemm.c:(.text+0x1e6): undefined reference to `sgemm_'
collect2: error: ld returned 1 exit status
make: *** [bin/im2rec] Error 1

**Comment by piiswrong** [comment]
What do you mean "I have put libblas.a libcblas.a to /usr/local/lib"? Where did they come from? Which blas library are you using?

**Comment by tqchen** [comment]
Was due to incomplete installation of blas, consider change blas version to for example open blas, might be the quickest way to solve the issue

**Comment by lhdgriver** [comment]
Hi, I have tried openblas, and fixed the problem. Thank you
